AppKit/CPColor.j

```diff
@@ -467,6 +467,16 @@ var cachedBlackColor,
     return [[CPColor alloc] _initWithCSSString: aString];
 }
 
++ (CPColor)selectedTextBackgroundColor
+{
+    return [CPColor colorWithHexString:"99CCFF"];
+}
+
++ (CPColor)_selectedTextBackgroundColorUnfocussed
+{
+    return [CPColor colorWithHexString:"CCCCCC"];
```

Comment:
Take a look to the file `AppKit/Themes/Aristo2/ThemeDescriptors.j`, line 104, you have :

```

+ (CPColor)themedColor
{
    var color = [CPColor redColor],
        themedColorValues =
    [
        [@"alternate-selected-control-color", [[CPColor alloc] _initWithRGBA:[0.22, 0.46, 0.84, 1.0]]],
        [@"secondary-selected-control-color", [[CPColor alloc] _initWithRGBA:[0.83, 0.83, 0.83, 1.0]]]
    ];

    [self registerThemeValues:themedColorValues forObject:color];

    return color;
}
```
This is the way you define a color for the theme Aristo2. You need to add two new color here, one for the `selectedTextBackgroundColor` and one for the `selectedTextBackgroundColor`.

Then, in `CPColor.j`, line 470 is the perfect example how to access the color defined in the theme :
```
+ (CPColor)alternateSelectedControlColor
{
    return [[self _cachedThemeColor] valueForThemeAttribute:@"alternate-selected-control-color"];
}
```

You will need to do something similar, but with a different key than `@"alternate-selected-control-color"`